### DISCUSSION TITLE
Any updates for broken API?

### DISCUSSION THREAD

**Discussion by sdnwser** [weather-gov/api#531] [General]
My API requests are now showing forecast information from 12/27/2021 being received on 2/19/2022.  Is there any progress on fixing this issue?   Will NWS provide any workarounds?
Thanks!

**Comment by chaunceygardiner** [comment]
There are two workarounds in this discussion forum.  The easiest is just to use preview-api.weather.gov instead of api.weather.gov  You’ll need to pay attention on this board as they will turn down preview-api.weather.gov at some point.

**Reply by sdnwser** [reply]
Oh nice, that works.  Thanks for the update.
